releasenotes/notes/0.4/0.4-summary-3de98711c3b7aa09.yaml

```diff
@@ -0,0 +1,95 @@
+---
+prelude: >
+    This release primarily focuses on tooling for simulating
+    `Qiskit Pulse <https://qiskit.org/documentation/apidoc/pulse.html>`__ schedules. The major
+    changes related to this in this release are:
+      - This release adds the :class:`.DynamicsBackend` class, which provides a
+        :class:`~qiskit.providers.backend.BackendV2` interface for running simulations. Once
+        constructed and configured, the :meth:`.DynamicsBackend.run` method can be used to simulate
+        schedules or circuits at the pulse level, returning measurement results in the same way as a
+        real backend. See the new :ref:`Simulating backends at the pulse-level with DynamicsBackend
+        <dynamics backend>` tutorial for how to set up the class for different applications.
+      - As of ``qiskit-terra`` 0.23.0, the pulse class
```

Comment:
This reads a bit off to me in the dynamics release notes prelude. I would word this to highlight the new dynamics converter more and at the end mention it requires qiskit-terra>=0.23.0. Or did the minimum terra version get bumped so people can't install dynamics 0.4.0 without terra 0.23.0.

Reply:
Hmm this is a good point. I've moved the sentence to the end. I've also now increased the minimum terra version to `0.23.0`. It's not critical functionality, but there are some other places too that we're using terra a little bit more (probability computation for quantum states) and also depends on more recent releases.